src/core/audio/AudioSdl.cpp

```diff
@@ -283,24 +304,98 @@ void AudioSdl::sdlInputAudioCallback(Uint8 *_buf, int _len) {
 
 #endif
 
+QString AudioSdl::setupWidget::s_systemDefaultDevice = QObject::tr("[System Default]");
```

Comment:
Is `QObject` the best context for this translation? I might opt for `AudioDeviceSetupWidget` instead (since `AudioSdl::setupWidget` doesn't currently have its own context, and this particular string may well end up shared among backends anyway).

Reply:
Thanks for checking! Fixed with commit https://github.com/LMMS/lmms/pull/7421/commits/f78e147d702253313fbce71183beb533f8aa472c.